### ISSUE TITLE
Push notifications on mobile

### ISSUE THREAD

**Issue by en0** [keybase/keybase-issues#3098]
As a user, i do not want to get push notifications on my mobile phone when actively chatting with another user in the desktop client. 

With the window open, in focus, and cursor in the input box: I still receive push notifications on my phone each time the person on the other end of the conversation sends a message. After that conversation, i had about 200 notifications to clear on my phone.  To clarify, i would still like to receive push notifications if i am not active on the chat if this is possible.

**Comment by DeedleFake** [comment]
As an addendum, at least on Android, getting a seperate notification for every message is quite annoying. Most chat apps have one notification that gets added to. The notifications don't even contain the message. They just say something along the lines of 'You have received a chat message.'

**Comment by junderw** [comment]
LINE, the most famous chat app in Japan also does 1 notification per message... I've gotten used to it >.<

But it would be nice if we had the option to change that.

**Comment by Ashaman-** [comment]
I agree with @DeedleFake notifications should be grouped together and include message content if phone is unlocked
